openid-4-verifiable-presentations-1_0.md

```diff
@@ -948,6 +948,23 @@ Wallet and/or the End-User if the value matching request
 is followed. Therefore, Verifiers MUST treat restrictions expressed using `values` as a
 best-effort way to improve user privacy, but MUST NOT rely on it for security checks.
 
+If a Wallet implements value matching and the credential to match against is
```

Comment:
The text is fine, but this should be moved below the `values` definition in Line 917. The processing rules in the current section define details on selecting claims and credentials, it's not a perfect fit here.